### ISSUE TITLE
It seems no internet for app for trollstore2.0.5

### ISSUE THREAD

**Issue by zzzwt** [opa334/TrollStore#580]
### No Duplicates Issue

- [X] I'm sure I've searched on the issue tracker before creating one.

### Expected Behavior?

It seems no internet for app for trollstore2.0.5

### Current Behavior?

app  running ok,but can not  access to internet

### Possible Solution?

-

### Steps to Reproduce

-

### TrollStore Version

TrollInstaller2.0.5

### iOS/iPadOS version

IOS16.1

### iDevice Model

iphone 13

**Comment by maxueli** [comment]
Yes, the same problem for iphone 14 promax IOS16.5, can't install from URL, install IPA File installation prompt can't access the internet

**Comment by songtao-yang** [comment]
same issue on iphone 14 promax 16.1.2, no network access alert after new installed by troll.

**Comment by lyl12321** [comment]
This may be related to the proxy. If you use a tun mode proxy or do not use a proxy, the software cannot access the Internet.This is what I tested.

**Comment by lyl12321** [comment]
I found that you can first switch to user mode, then enable network permissions in the system settings, and then switch to system mode to use it.

**Comment by zyu0090** [comment]
> I found that you can first switch to user mode, then enable network permissions in the system settings, and then switch to system mode to use it.

How to do this to TrollStore.app?

**Comment by shshbuxi** [comment]
> > I found that you can first switch to user mode, then enable network permissions in the system settings, and then switch to system mode to use it.
> 
> How to do this to TrollStore.app?

You can switch them in proxy app.

**Comment by wehnes** [comment]
> I found that you can first switch to user mode, then enable network permissions in the system settings, and then switch to system mode to use it.

I tried to change to user mode and the app settings appeared in system settings. But there is no network permission there. Only siri and choose language. Is there any way to solve this problem?

**Comment by shshbuxi** [comment]
> > I found that you can first switch to user mode, then enable network permissions in the system settings, and then switch to system mode to use it.
> 
> I tried to change to user mode and the app settings appeared in system settings. But there is no network permission there. Only siri and choose language. Is there any way to solve this problem?

The network permissions will be shown when I open connect bottom and switch global routing to direct in Shadowrocket.

**Comment by shshbuxi** [comment]
> > I found that you can first switch to user mode, then enable network permissions in the system settings, and then switch to system mode to use it.
> 
> I tried to change to user mode and the app settings appeared in system settings. But there is no network permission there. Only siri and choose language. Is there any way to solve this problem?

First, I open connect bottom and switch global routing to direct in Shadowrocket.
Then, the network permissions will be shown when I open the app which is installed by Trollstore.
